group.py

```diff
@@ -2,4 +2,8 @@
 
 # Your code to go here...
 
-my_group =
+my_group = {
+    "Jill": {"age": 26, "job": "biologist", "relation": {"friend": "Zalika", "Partner": "John"}},
+    "Zalika": {"age": 28, "job": "artist", "relation": {"friend": "Jill"}},
+    "John": {"age": 27, "job": "writer", "relation": {"partner": "Jill"}},
+    "Nash": {"age": 34, "job": "chef", "relation": {"cousin": "John", "landlord": "Zalika"}}
```

Comment:
[Suggestion] Maybe add a list for the job key value so it can be empty (for people without a job).  Well done, though!